### PR TITLE
fix bug for bool key type in neuralynxrawio.py

### PR DESCRIPTION
**Pull request by sin-mike**
in header there is a key:

```
-InputInverted False
```

which is interpreted as True when you use `bool('False')`

### CODE REVIEW

**Comment by samuelgarcia** [comment]
Oups. I am the guilty here.
Thank you for discover.

To stay more elegant I would add a func:

```python

...
    ('InputInverted', 'input_inverted', to_bool),  # used

...

    for k1, k2, type_conv in txt_header_keys:
        pattern = '-(?P<name>' + k1 + ') (?P<value>[\S ]*)'
        matches = re.findall(pattern, txt_header)
        for match in matches:
            if k2 == '':
                name = match[0]
            else:
                name = k2
            value = match[1].rstrip(' ')
            if type_conv is not None:
                value = type_conv(value)
            info[name] = value

...
def to_bool(txt):
    if txt=='True': 
        return True
    elif txt=='False': 
        return False
    else: 
        raise Exception('Connot convert %s to bool' %(txt))

```

**Comment by sin-mike** [comment]
> 
> 
> Oups. I am the guilty here.
> Thank you for discover.
> 
> To stay more elegant I would add a func:
> 
> ```python
> ...
>     ('InputInverted', 'input_inverted', to_bool),  # used
> 
> ...
> 
>     for k1, k2, type_conv in txt_header_keys:
>         pattern = '-(?P<name>' + k1 + ') (?P<value>[\S ]*)'
>         matches = re.findall(pattern, txt_header)
>         for match in matches:
>             if k2 == '':
>                 name = match[0]
>             else:
>                 name = k2
>             value = match[1].rstrip(' ')
>             if type_conv is not None:
>                 value = type_conv(value)
>             info[name] = value
> 
> ...
> def to_bool(txt):
>     if txt=='True': 
>         return True
>     elif txt=='False': 
>         return False
>     else: 
>         raise Exception('Connot convert %s to bool' %(txt))
> ```

yep, fixed

File: neo/rawio/neuralynxrawio.py [hunk 1]
```diff
@@ -490,6 +490,15 @@ def read_ncs_files(self, ncs_filenames):
...
 
 
+### Keys funcitons
+def _to_bool(txt):
+    if txt=='True': 
+        return True
+    elif txt=='False': 
+        return False
+    else: 
+        raise Exception('Connot convert %s to bool' %(txt))
```
Comment (JuliaSprenger) [inline neo/rawio/neuralynxrawio.py]: There's a typo in 'Can not'

**Comment by sin-mike** [comment]
> 
> 
> OK for merging for me.

yay!

**Pull request merged**